modules/virt-about-deploying-templates-custom-namespace.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-about-deploying-templates-custom-namespace.adoc
+
+[id="virt-about-deploying-templates-custom-namespace_{context}"]
+= About deploying templates to a custom namespace
+
+To add templates to a custom namespace, edit the `hco` custom resource (CR), add  `commonTemplatesNamespace` to the spec, and specify the Red Hat templates for that custom namespace.
```

Comment:
> specify the Red Hat templates for that custom namespace.

specify the custom namespace for the Red Hat templates.

Reply:
Updated the content.